### DISCUSSION TITLE
Future Roadmap/Plans/Ideas

### DISCUSSION THREAD

**Discussion by tonydero** [Mad-Cookies-Studio/mad-productivity#7] [Ideas]
Hello! Do you have any place listing the roadmap for this project or anything like that?

For instance, I noticed there's a "Game" tab with "TBD". What are your plans with that?

Also, will there eventually be (or maybe there already is but I'm missing it) a way to see a breakdown of total time for each time tracking reason/task?

Thanks! This is a very intriguing project!

**Comment by lentsius-bark** [comment]
Hello hello there! Truth be told we don't have any such a thing. So far the app has grown organically in its direction. o you have suggestions for where an how to roadmap could planned? Thanks!

**Reply by tonydero** [reply]
Good question! Maybe the GitHub Projects page? https://github.com/Mad-Cookies-Studio/mad-productivity/projects

**Comment by patronn44** [comment]
Write a commant
